packages/firestore/src/core/types.ts

```diff
@@ -26,6 +26,11 @@ export type BatchId = number;
  */
 export type TargetId = number;
 
+/**
+ * A randomly-generated key assigned to each Firestore instance at startup.
+ */
+export type InstanceKey = string;
```

Comment:
It's part of the public interface in this class and will eventually be used to create values of this type. I don't think it will ever be used without pulling in the multi-tab code, so I I'm ok with moving it there.